### PR TITLE
Create issue templates

### PR DESCRIPTION
**Pull request by atobiszei**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/ovms-bug-report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Logs**
+Logs from OVMS, ideally with --log_level DEBUG. Logs from client.
+
+**Configuration files**
+OVMS config.json file + model or publicly available similar model that reproduces the issue
```
Comment (atobiszei) [inline .github/ISSUE_TEMPLATE/ovms-bug-report.md:27]:
```suggestion
**Configuration files**
1. OVMS config.json file
2. Model repository directory structure
3. Model or publicly available similar model that reproduces the issue
```

File: .github/ISSUE_TEMPLATE/ovms-bug-report.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Steps to prepare models repository '...'
+2. OVMS launch command '....'
+3. Client command '....'
```
Comment (mzegla) [inline .github/ISSUE_TEMPLATE/ovms-bug-report.md:17]: If it's not our demo or sample then user could paste the client code

File: .github/ISSUE_TEMPLATE/ovms-bug-report.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Logs**
+Logs from OVMS, ideally with --log_level DEBUG. Logs from client.
+
+**Configuration files**
```
Comment (mzegla) [inline .github/ISSUE_TEMPLATE/ovms-bug-report.md:26]: Some machine and inference device details might be useful?
Docker image used (but I suppose we would get that from OVMS launch command)?

File: .github/ISSUE_TEMPLATE/ovms-bug-report.md (hunk 2 above)
Comment (atobiszei) [inline .github/ISSUE_TEMPLATE/ovms-bug-report.md:17]:
```suggestion
3. Client command (additionally client code if not using official client or demo) '....'
```

File: .github/ISSUE_TEMPLATE/ovms-bug-report.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+1. OVMS config.json file
+2. Model repository directory structure
+3. Model or publicly available similar model that reproduces the issue
```
Comment (atobiszei) [inline .github/ISSUE_TEMPLATE/ovms-bug-report.md:29]:
```suggestion
**Configuration**
1. OVMS version
2. OVMS config.json file
3. CPU, accelerator's versions if applicable
4. Model repository directory structure
5. Model or publicly available similar model that reproduces the issue

File: .github/ISSUE_TEMPLATE/ovms-bug-report.md (hunk 3 above)
Comment (atobiszei) [reply .github/ISSUE_TEMPLATE/ovms-bug-report.md]: I added request for HW. Yes we should get the docker image from the launch command. I want to keep this list short and simple.

**Review by mzegla — APPROVED** [review]

**Review by dkalinowski — APPROVED** [review]

**Pull request merged**